packages/subgraph/schema.graphql

```diff
@@ -318,7 +318,15 @@ type TokenVotingPlugin implements IPluginInstallation @entity {
   minProposerVotingPower: BigInt
   proposalCount: BigInt
   token: Token
-  members: [TokenVotingVoter!]! @derivedFrom(field: "plugin")
+  voters: [TokenVotingVoter!]! @derivedFrom(field: "plugin")
```

Comment:
You can decide if `voters` make sense anymore. the `voters` currently(it was called members previously) contain all the people that voted at least once on any proposal. This is how it was figuring out members. but since we have a different way now, not sure if such `voters` being on the `TokenVotingPlugin` schema makes sense at all. I think it should be in the `TokenVotingProposal` schema. I will leave it up to you.

@brickpop @Rekard0

Reply:
Voters relates to those who have voted. Members relates to those who **can vote**, totally different things